communities/epfl-hackers/community-assessment.md

```diff
@@ -0,0 +1,87 @@
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in working on cool tech projects outside of academia at EPFL.
```

Comment:
Solid! 
Things to think about:
- Your definition is very open, does your community appear that open from the outside? 
- Would someone from outside CS feel welcome? 
- Are non-code projects welcome?